Lib/test/test_typing.py

```diff
@@ -1610,6 +1610,41 @@ class P(Protocol):
         with self.assertRaisesRegex(TypeError, "@runtime_checkable"):
             isinstance(1, P)
 
+    def test_runtime_issubclass_with_classvar_data_members(self):
+        @runtime_checkable
+        class P(Protocol):
+            x: ClassVar[int] = 1
+
+        class C: pass
+
+        class D:
```

Comment:
Let's also add a case with the same class prop, but different value. Example:

```python
class E:
    x = 2
```

Because right now all names / values always match. It is not clear whether value is a part of the protocol or not.

Reply:
Agreed.

> It is not clear whether value is a part of the protocol or not.

It shouldn't be. IMO, we should only care about the "shape". Trying to runtime-check the value is difficult and should be left to third-party libs to do.